src/main/resources/fabric.mod.json

```diff
@@ -2,14 +2,16 @@
   "schemaVersion": 1,
   "id": "${id}",
   "version": "${version}",
-
   "name": "FabLabs ${libname} version ${libver}",
   "icon": "assets/${id}/icon.png",
   "description": "Fluid stack reference implementation for Fabric API",
   "license": "MIT",
   "contact": {
     "sources": "https://github.com/${repo}"
   },
+  "mixins": [
+    "fabric.fluids.mixin.json"
```

Comment:
Maybe give the mixin a better name? Like `fluidapi.mixins.json`

Also make the name plural.

Reply:
I prefer `mixin.json`, because the file is a mixin config, not just some list of the mixins in a package

Reply:
You didn't resolve my first request. `fabric.fluids` is quite vague. Hence match the name of the project more closely.